internal/openfga/openfga.go

```diff
@@ -308,6 +310,38 @@ func (o *OFGAClient) RemoveApplicationOffer(ctx context.Context, offer names.App
 	return nil
 }
 
+// RemoveRole removes a group.
```

Comment:
The docstrings and comments need to be updated.

Reply:
Also, how come there is no AddRole method?

Reply:
We appear to be just using auth relation CLI wise, there's no explicit relation command? Unless I'm mistaken? In our remove though, we clean up the group. I wanted to keep these the same.

Reply:
Took another look, makes sense, this function is just cleaning up as you said. AddGroup in this file doesn't even make sense, you'd add a person to a group or a group to something, not just AddGroup.